version.txt

```diff
@@ -8,9 +8,9 @@ COPYRIGHT_YEARS 2005-2021
 WEBSITE http://kodi.tv
 VERSION_MAJOR 21
 VERSION_MINOR 0
-VERSION_TAG BETA1
-VERSION_CODE 20.90.801
-ADDON_API 20.90.801
+VERSION_TAG BETA2
+VERSION_CODE 20.90.821
```

Comment:
Not 802?

Reply:
No, android shenanigans

Reply:
lol understanding is 0% but solidarity is 100%